From 5ceb7d3838d4c4316f10e034edae2f470e153c21 Mon Sep 17 00:00:00 2001
From: Rob Evers <revers@redhat.com>
Date: Wed, 25 Jun 2014 14:11:44 -0400
Subject: [scsi] be2iscsi: Bump the driver version

Message-id: <1403705504-27660-11-git-send-email-revers@redhat.com>
Patchwork-id: 84801
O-Subject: [RHEL7.1 e-stor PATCH 10/10] scsi: be2iscsi: Bump the driver version
Bugzilla: 1088576
RH-Acked-by: Tony Camuso <tcamuso@redhat.com>
RH-Acked-by: Tomas Henzl <thenzl@redhat.com>
RH-Acked-by: Maurizio Lombardi <mlombard@redhat.com>

From: John Soni Jose <sony.john-n@emulex.com>

http://marc.info/?l=linux-scsi&m=139934052304548&w=2

Signed-off-by: John Soni Jose <sony.john-n@emulex.com>
Signed-off-by: Jayamohan Kallickal <jayamohan.kallickal@emulex.com>
Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/drivers/scsi/be2iscsi/be_main.h b/drivers/scsi/be2iscsi/be_main.h
index 240c190..212ed3d 100644
--- a/drivers/scsi/be2iscsi/be_main.h
+++ b/drivers/scsi/be2iscsi/be_main.h
@@ -36,7 +36,7 @@
 #include <scsi/scsi_transport_iscsi.h>
 
 #define DRV_NAME  "be2iscsi"
-#define BUILD_STR  "10.2.125.0r"
+#define BUILD_STR  "10.2.273.0r"
 #define BE_NAME   "Emulex OneConnect" \
     "Open-iSCSI Driver version" BUILD_STR
 #define DRV_DESC  BE_NAME " " "Driver"
-- 
1.7.1